test/functional/api/v2/compute_resources_controller_test.rb

```diff
@@ -230,6 +235,19 @@ def teardown
     assert_equal storage_domain.id, available_storage_domains['results'].first.try(:[], 'id')
   end
 
+  test "should get specific vmware storage pod" do
+    storage_domain = Object.new
+    storage_domain.stubs(:name).returns('test_vmware_cluster')
+    storage_domain.stubs(:id).returns('group-p123456')
+
+    Foreman::Model::Vmware.any_instance.expects(:available_storage_pods).with('test_vmware_cluster').returns([storage_domain])
+
+    get :available_storage_pods, { :id => compute_resources(:vmware).to_param, :storage_pod => 'test_vmware_cluster' }
+    assert_response :success
+    available_storage_domains = ActiveSupport::JSON.decode(@response.body)
```

Comment:
This variable should be `available_storage_pods`, Rubocop it's complaining it's not being used & the test is failing